src/jenkins_job_insight/main.py

```
@@ -3811,6 +3813,117 @@ async def rotate_key_endpoint(request: Request, username: str) -> JSONResponse:
     )
 
 
+# -- Job Metadata Endpoints ---------------------------------------------------
+
+
+@app.get("/api/jobs/metadata")
+async def list_jobs_metadata(
+    team: str = Query(default=""),
+    tier: str = Query(default=""),
+    version: str = Query(default=""),
+    label: list[str] = Query(default=[]),
```

Ruff (0.15.10)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)